frontend/src/app/core/services/spinner.service.ts

```diff
@@ -5,9 +5,9 @@ import { BehaviorSubject } from 'rxjs';
 export class SpinnerService {
     private isLoading$$ = new BehaviorSubject<boolean>(false);
 
-    isLoading$ = this.isLoading$$.asObservable();
+    public isLoading$ = this.isLoading$$.asObservable();
 
-    show = () => this.isLoading$$.next(true);
+    public show = () => this.isLoading$$.next(true);
 
-    hide = () => this.isLoading$$.next(false);
+    public hide = () => this.isLoading$$.next(false);
```

Comment:
do we have a rule to add public to all public methods?

Reply:
I thought it would add some readability, because some public properties (without identifier) I changed to private

Reply:
Additionaly, I found a rule that all private fields in Angular must be alphabetical. Should I change this?

Reply:
In Angular, it is common practice to omit the "public" keyword when defining public methods. Since methods in Angular are public by default, explicitly using the "public" keyword is considered unnecessary and can be seen as redundant. Therefore, it is recommended to write your code without including "public" for public methods in accordance with common Angular style guidelines.